api/v1alpha1/etcd_types.go

```diff
@@ -230,6 +230,15 @@ type EtcdConfig struct {
 	// HeartbeatDuration defines the duration for members to send heartbeats. The default value is 10s.
 	// +optional
 	HeartbeatDuration *metav1.Duration `json:"heartbeatDuration,omitempty"`
+	// ClientService defines specified annotations that should be added to the client service
```

Comment:
Can you please check this doc string? I think it's not generic enough if we want to expand the `ClientService` struct in the future, i.e. don't mention annotations here.